CodePush.m

```diff
@@ -351,9 +386,7 @@ - (void)savePendingUpdate:(NSString *)packageHash
             [self savePendingUpdate:updatePackage[PackageHashKey]
                           isLoading:NO];
             
-            if (installMode == CodePushInstallModeImmediate) {
-                [self loadBundle];
-            } else if (installMode == CodePushInstallModeOnNextResume) {
+            if (installMode == CodePushInstallModeOnNextResume) {
```

Comment:
Yes it would, I am currently working on android changes now, will incorporate all of yours and shishir's feedback soon.